crates/vm/levm/src/opcodes.rs

```diff
@@ -368,14 +369,11 @@ impl<'a> OpCodeFn<'a> {
 }
 
 impl<'a> VM<'a> {
-    /// The opcode table mapping opcodes to opcode handlers for fast lookup.
-    pub(crate) const OPCODE_TABLE: [OpCodeFn<'a>; 256] = VM::build_opcode_table();
```

Comment:
Dropping the const is not really necessary.

Reply:
what do you mean? i have to mutate the table at runtime according to the fork

Reply:
I agree with this. We have a set number of forks. I think we should build a const opcode table for each fork at the beginning of execution, and then at VM build time we should just match over the fork and get a pointer to the appropriate op code table. What do you think?